.github/scripts/trymerge.py

```diff
@@ -747,9 +770,24 @@ def merge_into(self, repo: GitRepo, *, force: bool = False, dry_run: bool = Fals
         else:
             self.merge_ghstack_into(repo, force, comment_id=comment_id)
 
-        repo.push(self.default_branch(), dry_run)
-        if not dry_run:
-            gh_add_labels(self.org, self.project, self.pr_num, ["merged"])
+    def create_land_time_check_branch(self,
+                                      repo: GitRepo,
+                                      branch: str,
+                                      force: bool = False,
+                                      comment_id: Optional[int] = None,) -> str:
+        self.merge_changes(repo, branch=branch, force=force, comment_id=comment_id)
+        land_check_branch = f'landchecks/{self.pr_num}'
+        try:
+            repo._run_git('branch', "-D", land_check_branch)
```

Comment:
Do we actually want to do this? If we delete branches, will people then not be able to see their previous land attempts? If they can still see them, I'd be ok with this.

Reply:
The commit still exists so you can still look at it if you want in HUD or in GH. branch will just be gone I guess which is fine.